blackhole.py

```diff
@@ -333,34 +346,40 @@ async def is_accessible(path, timeout=10):
             torrentConstructors = []
             if realdebrid['enabled']:
                 torrentConstructors.append(RealDebridTorrent if file.torrentInfo.isDotTorrentFile else RealDebridMagnet)
+                debrid_provider = "Real-Debrid"
             if torbox['enabled']:
                 torrentConstructors.append(TorboxTorrent if file.torrentInfo.isDotTorrentFile else TorboxMagnet)
+                debrid_provider = "Torbox"
```

Comment:
They can both be enabled, so this is probably something we should just pull directly from `TorrentBase` type